files/ru/web/html/element/a/index.md

```diff
@@ -67,18 +67,18 @@ HTML-элемент `<a>` определяет гиперссылку для п
 
 ### Устаревшие
 
-- {{htmlattrdef("charset")}} {{obsoleteGeneric("inline","HTML5")}}
+- {{htmlattrdef("charset")}} {{obsolete_inline}}
```

Comment:
I guess this should have been `{{obsolete_inline("html5")}}`, but it probably doesn't matter for these few occurrences.

Reply:
This got removed in #11689 entirely since both macros are deprecated anyways!